Adds more decorations!

OnsenCapy opened the pull request:
<!--
SPDX-FileCopyrightText: 2021 Pieter-Jan Briers <pieterjan.briers+git@gmail.com>
SPDX-FileCopyrightText: 2021 Swept <sweptwastaken@protonmail.com>
SPDX-FileCopyrightText: 2021 mirrorcult <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2022 AJCM-git <60196617+AJCM-git@users.noreply.github.com>
SPDX-FileCopyrightText: 2022 Kara <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2023 DrSmugleaf <DrSmugleaf@users.noreply.github.com>
SPDX-FileCopyrightText: 2023 Kevin Zheng <kevinz5000@gmail.com>
SPDX-FileCopyrightText: 2024 Vasilis <vasilis@pikachu.systems>
SPDX-FileCopyrightText: 2024 lzk <124214523+lzk228@users.noreply.github.com>
SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>

SPDX-License-Identifier: AGPL-3.0-or-later
-->

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- NOTE: All code submitted to this repository is ALWAYS licensed under the AGPL-3.0-or-later license. 
The REUSE Specification headers or separate .license files indicate a secondary license (e.g., MPL or MIT) solely to facilitate 
integration for projects that do not use the AGPL license. This secondary license does not replace the fact that AGPL-3.0-or-later remains the primary and binding license. 
Uncomment and modify the following line if you wish to change the license from the default of AGPL.-->
<!--- LICENSE: AGPL -->
## About the PR
Added several new objects, such as:
Ceiling fans!
Candelabras!
Wall clocks!
Flower vases!
Quills!
Desktop Computers!
Barrels!
Construction Barricades!
Ant hills!
Garbage!
A dead god! (Idk, might be useful for something, eventually)

## Why / Balance
There is a severe lack of objects that can be used for mapping to differentiate one area from the other. The objects added also aim to improve ambience by adding new sounds to the game.

## Media

https://github.com/user-attachments/assets/f9520dbb-652c-4cc9-9f73-5e459c303b43

https://github.com/user-attachments/assets/6840a411-2a1f-46ba-982b-e06b4cfdb014

https://github.com/user-attachments/assets/c40b93c0-e9a8-496c-8df7-a928911b5ec8

Water was removed due to maintainer's request.

https://github.com/user-attachments/assets/80c32abf-6931-4dc1-ba73-4b763ff8e9fe


<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
:cl: OnsenCapy
- add: Added several new furniture to decorate the station with!

afuckingidiot commented:
does it make fan sounds

OnsenCapy commented:
> does it make fan sounds

You bet your ass it does! I started this PR so I could add some more ambience sounds to the station.

OnsenCapy commented:
I was incredibly tempted to just add several machines that do absolutely nothing but make ambience noise but I'm sure maintainers would yell at me give them a purpose.

Thatonestomf commented:
This is sick, we need more pr's like this
And if someone wants to make them do things they can afterwards

OnsenCapy commented:
> I'm personally against fans and candelabras, messes up the perspective, I _do not_ know how to make them look like they're "above" you. I'd just say it's a can of worms best left unopened 

I think our lack of ceiling objects really is detrimental to the immersion and map building scene. Inevitably there is going to be some mild art style clash by pulling sprites from other codebases, but I really think the ceiling fans in detective rooms and candelabras in chapels could really improve the general look.

> Barrels look fine aside from barrel fire, which doesn't exactly look part

Don't really agree, it should be fine. It is going to be used very sparsely anyway.

> Running water is IMO a no-go since it introduces issues like "floating" water, unless it's only placed on indestructible tiles. 
 
 Agreed and removed. The textures are originally from showerheads, but since I'm just a YAML warrior, I only added the water. The sprites does have a "dripping from the ceiling" state, but it is probably best implemented in some other way.
 
> don't we already have cones?

Funnily enough, we do. It just never occured to me, because it is called "warning cone", so it always shows up last in the list. Removed.

OnsenCapy commented:
Test fail not related

SX-7 reviewed (approved):
Bleh, if fans/chandelabras happen to be weird then they're just gonna be removed. IMO it's bad since we don't have ceilings, so they can just be floating midspace...
